src/core/tokens/example_color_tokens.scss

```diff
@@ -1,19 +1,23 @@
 :root {
-  --rm-color-white: hsl(0, 100%, 100%);
-  --rm-color-black: hsl(0, 0%, 0%);
+  --rm-color-white: hsl(0deg 100% 100%);
```

Comment:
@Jeremy-Walton Do we need to include this in the lint? It's not required to use `deg`.

Reply:
The linting only requires it when you pass 0. I think it's actually helpful to make it clear what is happening.